pkg/registry/core/pod/strategy_test.go

```diff
@@ -2337,6 +2337,7 @@ func TestPodLifecycleSleepActionEnablement(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))
```

Comment:
ditto. It isn't obvious to me why we would choose n-1 from GA promotion for emulated mode. Maybe I missed that memo though.